extension-manifest-v3/src/utils/engines.js

```diff
@@ -271,7 +271,11 @@ export async function update(name) {
     }
 
     // Make a full update if we need to remove some lists
-    if (foundListsToRemove) {
+    if (
+      foundListsToRemove ||
+      (name === 'trackerdb' &&
+        data.lists[0].checksum !== engine.lists[0].checksum)
```

Comment:
Currently, the trackerdb does not have any list - the code assumes that it exists, and will throw if there are no `lists[0]`. We must be sure that the "new" TrackerDB will be included in the release, or add

```suggestion
        data.lists[0] && data.lists[0].checksum !== engine.lists[0].checksum)
```

Reply:
Ok, it's even worse.. the `engine.lists` is a Map...